OrbitQt/servicedeploymanager.cpp

```diff
@@ -29,6 +29,19 @@ static const std::chrono::milliseconds kSshWatchdogInterval(1000);
 
 namespace OrbitQt {
 
+namespace {
+
+void PrintAsOrbitService(const std::string& buffer) {
+  std::vector<std::string_view> lines = absl::StrSplit(buffer, '\n');
+  for (const auto& line : lines) {
+    if (!line.empty()) {
+      LOG("[OrbitService] %s", line);
```

Comment:
I learned more about our logging infra and I think it's best to use LOG_PLATFORM here. This will avoid to print line and file for log messages coming from the service.

Reply:
Makes sense. Done (and made it even nicer looking with some inspiration from Logging.h)